### PR TITLE
Ruth ikegah/issue/#46

### LINKED ISSUE #46: [Docs] Contributing: website contributing instructions needed
**Current State:**
The CONTRIBUTING.md in this repo needs is missing some information and instructions specific to this project.

**Desired State:**
The CONTRIBUTING.md in this repo needs to be updated information about what framework and plugins that the GetNighthawk website uses. It also needs updated with instructions for how to contribute and how to build the GetNighthawk website.

Resource: See the [GetNightHawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w/edit#) for details

### PR DESCRIPTION
**Pull request by Ruth-ikegah**
**Description**
I updated this file with the Project description and how to contribute and build the site.

This PR fixes #46

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,15 +5,33 @@ All contributors are welcome. Please see the [newcomers welcome guide](https://d
...
+1. Visit the [Project documentation](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w/edit#) 
+2. Suggest requests, changes, or features by creating an issue on the repository
+3. Look at existing issues to tackle and get involved with
+
+
+## Steps to build GetNightHawk site
+
+1. `cd` into the getnighthawk repo
+2. `cd docs` to access the getnighthawk jekyll website codebase
+3. Run the following code in your terminal `Bundle install`.
```
Comment (leecalcote) [inline CONTRIBUTING.md:26]:
```suggestion
3. Run the following code in your terminal `bundle install`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,15 +5,33 @@ All contributors are welcome. Please see the [newcomers welcome guide](https://d
...
+4. Run `make site` in order to serve the jekyll website on your machine at `http://127.0.0.1:4000/`
+( `bundle exec jekyll serve` can also be used for spining up the jekyll website at `http://127.0.0.1:4000/`)
```
Comment (leecalcote) [inline CONTRIBUTING.md:28]:
```suggestion
( `bundle exec jekyll serve` can also be used for spinning up the Jekyll website at `http://127.0.0.1:4000/`)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,15 +5,33 @@ All contributors are welcome. Please see the [newcomers welcome guide](https://d
...
+## Steps to contribute to GetNightHawk project
+1. Visit the [Project documentation](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w/edit#) 
+2. Suggest requests, changes, or features by creating an issue on the repository
+3. Look at existing issues to tackle and get involved with
+
+
+## Steps to build GetNightHawk site
+
+1. `cd` into the getnighthawk repo
+2. `cd docs` to access the getnighthawk jekyll website codebase
```
Comment (leecalcote) [inline CONTRIBUTING.md:25]:
```suggestion
2. `cd docs` to access the GetNighthawk Jekyll website codebase
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -5,15 +5,33 @@ All contributors are welcome. Please see the [newcomers welcome guide](https://d
...
+
+## Steps to contribute to GetNightHawk project
+1. Visit the [Project documentation](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w/edit#) 
+2. Suggest requests, changes, or features by creating an issue on the repository
+3. Look at existing issues to tackle and get involved with
+
+
+## Steps to build GetNightHawk site
+
+1. `cd` into the getnighthawk repo
```
Comment (leecalcote) [inline CONTRIBUTING.md:24]: Perhaps, this first step should be to fork the repo.

**Review by leecalcote — COMMENTED** [review]
Here's a helpful reference - https://github.com/layer5io/meshery/tree/master/docs

**Comment by Ruth-ikegah** [comment]
@leecalcote I will fix this up in the other PR now! DCO check messing with me here lol

**Pull request closed without merging**